addon/components/frost-toggle.js

```diff
@@ -40,27 +52,30 @@ export default Component.extend(FrostEventsProxy, {
   // == Functions ==============================================================
   init () {
     this._super(...arguments)
-    Ember.assert(`Same value has been assigned to both ${this.toString()}.trueValue and ${this.toString()}.falseValue`,
+    assert(`Same value has been assigned to both ${this.toString()}.trueValue and ${this.toString()}.falseValue`,
       (typeof this.attrs['trueValue'] === 'undefined' && typeof this.attrs['falseValue'] === 'undefined') ||
       this.attrs['trueValue'] !== this.attrs['falseValue'])
   },
 
   getDefaultProps () {
```

Comment:
shouldnt getDefaultProps be in the above section, just under the prop types declaration? so that will be in the state properties section.
And init() would be under `Events` section for now since its not one of our functions. The name will change to `Lifecycle hooks` at some point.